ckan/controllers/package.py

```diff
@@ -165,7 +165,8 @@ def remove_field(key, value=None, replace=None):
 
         c.remove_field = remove_field
 
-        sort_by = request.params.get('sort', None)
+        sort_by = request.params.get('sort', 'metadata_modified desc')
+        request.GET['sort'] = sort_by
```

Comment:
maybe should be ` if sort_by==None: request['GET']= sort_by= 'metadata_modified desc' ` is more clear?